server/server_test.go

```diff
@@ -105,6 +105,9 @@ func TestTCPv4Network(t *testing.T) {
 
 		require.NoError(t, srv.grpcListener.Close())
 		require.NotNil(t, <-errChan)
+
+		// So that address is freed for further tests.
+		srv.Shutdown()
```

Comment:
Is this a linter-driven change?

Reply:
No - I was running tests multiple times with `go test github.com/weaveworks/common/server -count 10` and ran into few issues. I've reverted this now, so that linter commit only contains linter changes.